dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -241,7 +255,7 @@ def self.get_courses(state_code, row_hash)
       CA_COURSE_CODES.select {|course| course == row_hash['CourseCode']}
     when 'CT'
       enrollment = row_hash['CourseEnrollments']
-      # Don't consider a course as offered at a school if there is no enrollment ("*") and it is a positive number
+      # Don't consider a course as offered at a school if there is no enrollment ("*") or it is not a positive number
       CT_COURSE_CODES.select {|course| course == row_hash['Course'] && enrollment != '*' && enrollment.to_i > 0}
```

Comment:
Any issue if `enrollment.to_i == 0`?

Reply:
I talked to @mirlew and she suggested we treat no enrollment ("*" or "0") as the course not being offered.